main/bamboo/bamboo.go

```diff
@@ -105,7 +106,12 @@ func initServer(conf *configuration.Configuration, conn *zk.Conn, eventBus *even
 	// Static pages
 	router.Use(martini.Static(path.Join(executableFolder(), "webapp")))
 
-	registerMarathonEvent(conf)
+	if conf.Marathon.UseEventStream {
+		// Listen events stream from Marathon
+		listenToEventStream(conf, eventSubAPI)
```

Comment:
minor consistency issue. Do you think i's better to align with the callback model function name? e.g. renaming `listenToEventSteam` to `registerMarathonEventStream`